File: android/BertSpeak/app/src/main/java/chuckcoughlin/bert/tab/AssistantFragment.java
/**
 * Copyright 2019 Charles Coughlin. All rights reserved.
 *  (MIT License)
 */
package chuckcoughlin.bert.tab;


import android.os.Bundle;
import android.support.v4.app.Fragment;

/**
 * This interface is required for all fragments in the application.
 */

public interface AssistantFragment {

    String PAGE_ARG = "page";
    String TITLE_ARG= "title";

    int getPageNumber();
    void setPageNumber(int page);
    String getTitle();
    void setTitle(String title);

}

File: BertSpeak/root/src/main/kotlin/chuckcoughlin/bertspeak/tab/AssistantFragment.kt
/**
 * Copyright 2022 Charles Coughlin. All rights reserved.
 * (MIT License)
 */
package chuckcoughlin.bertspeak.tab

/**
 * This interface is required for all fragments in the application.
 */
interface AssistantFragment {
    val pageNumber: Int
    var title: String?

    companion object {
        const val PAGE_ARG  = "page"
        const val TITLE_ARG = "title"
    }
}
